docs/4-developer/README.md

```diff
@@ -1,6 +1,6 @@
 # Developer Manual
 
-The development repository is https://github.com/codepod-io/codepod
+The development repository is https://github.com/codepod-io/codepod. Please also check https://github.com/codepod-io/codepod/doc/onboarding.md for the development framework and system architecture description.
```

Comment:
Instead of adding a link to the `codepod.io/codepod` source code repo, I think we should put and maintain all `.md` files here so that they are rendered on our website.

Reply:
IIUC, 
1. codepod.io/docs/4-deverloper/README should sync with codepod-io/codepod/README.
2. Add the onboarding doc as one section in ../4-developer/README
3. Render the ../4-deverlop/README in codepod-io/codepod/README.

Reply:
> 1. codepod.io/docs/4-deverloper/README should sync with

Yes, these two should sync. But I would just remove the dev-setup part from `codepod-io/codepod/README`, but keep everything in `codepod.io/docs/4-deverloper/README`.

> 2. ..

Yes

> 3. 
Not render, but just put a link in `codepod-io/codepod/README`

I'll make this change.

Reply:
Now (1) and (3) should be done in #9. You'll be able to work on (2).

Reply:
> Now (1) and (3) should be done in #9. You'll be able to work on (2).

PR updated.